### PR TITLE
Mark some OIDC fields as sensitive

### PR DESCRIPTION
**Pull request by dmsergeevN26**
Mark both `client_secret` and `google_service_account` as sensitive to avoid secret leakage by terraform. Not sure if I need to add any tests for this. Please let me know if so.

### CODE REVIEW

**Comment by dmsergeevN26** [comment]
Actually, it might be an autogenerated file. Let me take  a deeper look. First time working with terraform sdk

**Comment by dmsergeevN26** [comment]
What was done:
1. Modified the OIDCConnectorV2 path as it changed at some point
2. Ran `make gen-schema`

**Comment by gzigzigzeo** [comment]
Hey! Thank you! LGTM.

File: terraform/tfschema/types_terraform.go [hunk 1]
```diff
@@ -1250,7 +1348,7 @@ func GenSchemaMetaProvisionTokenV2() map[string]*accessors.SchemaMeta {
 				},
 
 				"allow": {
-					Name:       "Allow",
+					Name:       "allow",
```
Comment (gzigzigzeo) [inline terraform/tfschema/types_terraform.go:1351]: Due to the inconsistency in `types.proto` this should be capitalized back manually.

**Review by gzigzigzeo — CHANGES_REQUESTED** [review]

**Comment by Joerger** [comment]
Thanks for you contribution @dmsergeevN26, I've opened an internal PR (with the changes requested) to get this change reviewed, tested through CI, and merged.

**Comment by dmsergeevN26** [comment]
@Joerger Thanks. I will close this one, then.

**Pull request closed without merging**